socorro/external/crashstorage_base.py

```diff
@@ -143,31 +136,18 @@ def migrate_raw_crash(data):
     return data
 
 
-class CrashStorageBase(RequiredConfig):
+class CrashStorageBase:
     """Base class for all crash storage classes."""
 
-    required_config = Namespace()
-
-    def __init__(self, config, namespace=""):
-        """Initializer
-
-        :param config: configman DotDict holding configuration information
-        :param namespace: namespace for this crashstorage instance; used
-            for metrics prefixes and logging
-
-        """
-        self.config = config
-        self.namespace = namespace
-
+    def __init__(self):
         # Collection of non-fatal exceptions that can be raised by a given storage
         # implementation. This may be fetched by a client of the crashstorge so that it
         # can determine if it can try a failed storage operation again.
-        self.exceptions_eligible_for_retry = ()
         self.logger = logging.getLogger(__name__ + "." + self.__class__.__name__)
+        self.exceptions_eligible_for_retry = ()
```

Comment:
I'm not sure why this line got moved, but we should move it back because the comment applies to it.